src/app/(authorized)/profile/[id]/PostFix/EditPost/EditPostMobile.tsx

```diff
@@ -0,0 +1,155 @@
+import { ChangeEvent, useState } from 'react';
+import { toast } from 'react-toastify';
+
+import { useUpdatePostMutation } from '@/api';
+import { Loader } from '@/components/Loader';
+
+import s from './EditPost.module.scss';
+import Cookies from 'js-cookie';
+import { useTranslation } from 'react-i18next';
+import { EditPostModal } from '@/components/Modals/EditPostModal';
+import Image from 'next/image';
+import { Carousel } from '@/components/Carousel/Carousel';
+import { ImageType } from '@/api/posts.api';
+import { SwiperSlide } from 'swiper/react';
+import { PostModal } from '@/components/Modals/PostModal';
+import { UserProfile } from '@/app/(authorized)/profile/[id]/types';
+
+type Props = {
+  setEditPost: (value: boolean) => void;
+  description: string;
+  postId: number | undefined;
+  user: UserProfile;
+  images: ImageType[];
+};
+
+export const EditPostMobile = ({
+                                 setEditPost,
+                                 description,
+                                 user,
+                                 images,
+                                 postId,
+                               }: Props) => {
+  const [textareaValue, setTextareaValue] = useState(description);
+  const { t } = useTranslation();
+  const translate = (key: string): string => t(`CreatePost.EditPost.${key}`);
+  const accessToken = Cookies.get('accessToken');
+
+  const [updatePost, { isLoading }] = useUpdatePostMutation();
```

Comment:
we are using server actions (all RTK Query are deprecated, If u want to use it , check if interceptor rtk query works)
I`d like to see server actions here